production/ksonnet/loki/bloom-compactor.libsonnet

```diff
@@ -0,0 +1,118 @@
+{
+  local k = import 'ksonnet-util/kausal.libsonnet',
+  local container = k.core.v1.container,
+  local containerPort = k.core.v1.containerPort,
+  local pvc = k.core.v1.persistentVolumeClaim,
+  local service = k.core.v1.service,
+  local statefulSet = k.apps.v1.statefulSet,
+  local volume = k.core.v1.volume,
+  local volumeMount = k.core.v1.volumeMount,
+
+  local name = 'bloom-compactor',
+
+  _config+:: {
+    bloom_compactor+: {
+      pvc_size: '256Gi',  // error 'define bloom_compactor.pvc_size',
+      pvc_class: 'fast',  // error 'define bloom_compactor.pvc_class',
+    },
+    loki+: {
+      bloom_compactor+: {
+        enabled: $._config.use_bloom_filters,
+        working_directory: '/data/blooms',
+        compaction_interval: '15m',
```

Comment:
we can get that `compaction_interval` from config too.

Reply:
we don't have the `compactor.compaction_interval` defined in the lib